<launch>
	<!-- logger params-->
	<arg name="trajectory_rate"	default="10"	doc="Ros_rate in Hz"/>
	<arg name="object_name"	default="gapter1"	doc="Object name to capture in relevant topics" />
	
	<!-- load the logging node -->
	<node pkg="flight_pkg" type="leader_follower_v3" name="leader_follower_v3_$(arg object_name)" output="screen" >
		<!-- Load parameters onto server using argument or default values above -->
		<param name="trajectory_rate"	value="$(arg trajectory_rate)" />
		<param name="object_name"	value="$(arg object_name)" />
		<param name="follower_distance"	type="double"	value="2.0"/>
		<param name="delay_time"	type="double"	value="1.25"/>
		<param name="pose_buffer"	type="double"	value="1.75"/>
	</node>
</launch>
